.github/workflows/ci-manual.yaml

```diff
@@ -141,16 +103,19 @@ jobs:
             };
             const uniqueStepNames = new Set();
             const stepConclusionsByJob = new Map();
-            const ciStepNamePrefix = "${{ needs.setup.outputs.ci_step_name_prefix }}";
+            const ciStepNamePrefix = "${{ needs.ci.outputs.ci_step_name_prefix }}";
             for (const job of data.jobs) {
-              if (/^\d+\.\d+/.test(job.name)) {
+              // look for jobs named "ci / k8sVersion / osDistro"
+              if (/^ci \/ \d+\.\d+/.test(job.name)) {
```

Comment:
this didn't change did it? the step name definition is still
```
    name: ${{ matrix.k8s_version }} / ${{ matrix.os_distro }}
```

Reply:
GitHub prefixes the step name with the reusable workflows name

Reply:
ah thanks, the regex and trimming feels sorta awkward but i tried perusing the api workflows response (https://api.github.com/repos/cartermckinnon/amazon-eks-ami/actions/runs/11411576922/jobs) and i can't really come up with a better plan either 🤷

Reply:
Yeah it gets the job done for now